src/main/java/usecase_gamedata/PlayerFactory.java

```diff
@@ -1,25 +1,32 @@
 package usecase_gamedata;
 
+import FileReader.playerFileReader;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import entity.BasicEquipmentSlots;
 import entity.CollectibleInventory;
 import entity.Player;
-import javax.json;
 
+public class PlayerFactory extends factory<Player>{
+    public Player getPlayer(int maxHP, int atkPT, CollectibleInventory inventory, BasicEquipmentSlots equipments,
+                            int[] location){
+        // I want to leave out Sword and entity.Armor because if we were to have a class system we would be more
+        // flexible on that. For example, have a new input of String class and for every class the factory
+        // gets the starter equipment and items for that specific class and build the player.
 
 
-public class PlayerFactory {
-    private PlayerBuilder builder;
-    private JSONObject object = Json.createObject
 
-    public Player create(JSONObject jsonObject, GameData gameData) {
-        Player player;
```

Comment:
I am very confused about this deletion. Could you explain this?

Reply:
GameFileReader no longer returns an jsonObject, it returns the string form of jsonObject. We use the string form of the jsonObject to make Player.